app/assets/stylesheets/show.scss

```diff
@@ -12,28 +14,35 @@
   margin-left: 10px;
   padding-left: 5px;
   p {
-    display: inline;
     padding-right: 5px;
   }
 }
 
 .voting-title-container {
-  padding-top: 20px;
+  margin: 0;
   vertical-align: middle;
+  background-color: lightgray;
```

Comment:
Color `lightgray` should be written in hexadecimal form as `#d3d3d3`<br>Color literals like `lightgray` should only be used in variable declarations; they should be referred to via variable everywhere else.